### DISCUSSION TITLE
Please add a connection option for the cloud storage service: Sync

### DISCUSSION THREAD

**Discussion by throwaway1908** [iterate-ch/cyberduck#15189] [Ideas]
Sync's website: https://www.sync.com/
Would really appreciate this option, as Sync is my main cloud storage service that I use.

**Comment by tatze96** [comment]
As far as I know, sync.com doesn't have a public API we could implement to support the storage service natively.
Additionally, based on their [FAQ](https://www.sync.com/help/general-limits-when-using-sync/#gnas), they don't support protocols like FTP(S), SFTP, or WebDAV which is why the storage isn't compatible with Cyberduck and Mountain Duck. We, therefore, can't add a connection option or connection profile for sync.com.
